src/components/claims/claim-details/index.tsx

```diff
@@ -137,6 +137,12 @@ const ClaimPage: React.FC<Props> = ({ ...props }) => (
                           reserves={reserves}
                           sanctionStatus={member.sanctionStatus!}
                           refetchPage={refetch}
+                          identity={member.identity ? member.identity : null}
+                          market={
+                            member.contractMarketInfo?.market
+                              ? member.contractMarketInfo?.market
+                              : null
+                          }
```

Comment:
This one is pretty neat
```suggestion
                          market={member.contractMarketInfo?.market ?? null}
```